pkg/frontend/openshiftclusterkubeconfigcredentials_post_test.go

```diff
@@ -0,0 +1,338 @@
+package frontend
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"strings"
+	"testing"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+	"github.com/Azure/ARO-RP/pkg/api/v20210131preview"
+	"github.com/Azure/ARO-RP/pkg/database/cosmosdb"
+	"github.com/Azure/ARO-RP/pkg/metrics/noop"
+	testdatabase "github.com/Azure/ARO-RP/test/database"
+)
+
+func TestPostOpenShiftClusterKubeConfigCredentials(t *testing.T) {
+	ctx := context.Background()
+
+	apis := map[string]*api.Version{
+		"2021-01-31-preview": api.APIs["2021-01-31-preview"],
+		"no-credentials": {
+			OpenShiftClusterConverter:       api.APIs["2021-01-31-preview"].OpenShiftClusterConverter,
+			OpenShiftClusterStaticValidator: api.APIs["2021-01-31-preview"].OpenShiftClusterStaticValidator,
+		},
+	}
+
+	mockSubID := "00000000-0000-0000-0000-000000000000"
+	resourceID := fmt.Sprintf("/subscriptions/%s/resourcegroups/resourceGroup/providers/Microsoft.RedHatOpenShift/openShiftClusters/resourceName", mockSubID)
+
+	type test struct {
+		name           string
+		resourceID     string
```

Comment:
In https://github.com/Azure/ARO-RP/pull/1499#discussion_r648265442 I meant that we can remove this field at all becuase all values are the same.

But again: It is just a nit. Not something that justifies blocking PR from merging.

Reply:
Lets skip this time :)